### PR TITLE
Server: add editor specific loading shell.

### PR DESCRIPTION
**Pull request by mtias**
screen shot 2017-03-22 at 13 22 18

### CODE REVIEW

File: server/pages/index.js [hunk 1]
```diff
@@ -354,6 +354,10 @@ module.exports = function() {
 						req.context.hasSecondary = true;
 					}
 
+					if ( section.group && req.context ) {
+						req.context.sectionGroup = section.group;
```
Comment (ehg) [inline server/pages/index.js]: At this point, we may as well pass the entire `section` object :P

Comment (mtias) [inline server/pages/index.js]: One piece at a time ;)

**Comment by jasmussen** [comment]
Love it! Looks good, works good. 

I see this at smaller breakpoints, though:

screen shot 2017-03-22 at 13 30 28

**Comment by mtias** [comment]
That seems like a bug of the mobile layout. I'll investigate separately.

**Pull request merged**